### PR TITLE
Document process for collecting crash dumps

### LINKED ISSUE #11210: Document process for collecting crash dumps
For AppServices we might want to use https://blogs.msdn.microsoft.com/asiatech/2015/12/28/use-crash-diagnoser-site-extension-to-capture-dump-for-intermittent-exception-issues-or-performance-issues-on-azure-web-app/ as a basis

For on-prem scenarios there is https://docs.microsoft.com/en-us/windows/desktop/wer/collecting-user-mode-dumps but we might want to turn it into exact set of steps with correct values or even a powershell script (https://github.com/aspnet/AspNetCore/blob/cb21edc5009b254388fc6da93d212f679087c326/.azure/pipelines/tools/SetupTestEnvironment.ps1#L71-L73).

cc @guardrex @shirhatti

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #11210

[Internal Review Topic (Slow or hanging app section)](https://review.docs.microsoft.com/en-us/aspnet/core/host-and-deploy/iis/troubleshoot?view=aspnetcore-1.0&branch=pr-en-us-11301#slow-or-hanging-app)

### CODE REVIEW

File: aspnetcore/host-and-deploy/iis/troubleshoot.md [hunk 1]
```diff
@@ -232,11 +232,52 @@ If an app is capable of responding to requests, obtain request, connection, and
 
 ## Slow or hanging app
 
-When an app responds slowly or hangs on a request, obtain and analyze a [dump file](/visualstudio/debugger/using-dump-files). Dump files can be obtained using any of the following tools:
+When an app responds slowly or hangs on a request, obtain and analyze a *crash dump* from [Windows Error Reporting (WER)](/windows/desktop/wer/windows-error-reporting). A crash dump contains a copy of the system's memory at the time of an app crash and can help determine the cause of the crash.
```
Comment (pakrym) [inline aspnetcore/host-and-deploy/iis/troubleshoot.md]: This only applies when the application actually crashes. To be able to get a dump of an app that's just slow you need to use something different then WER.

**Comment by guardrex** [comment]
@pakrym ... :eyes: how about this?

**Comment by pakrym** [comment]
Looks great. Let's give @shirhatti couple days to take a look.

**Comment by guardrex** [comment]
Ping @shirhatti ... it's a quick review on this one.

**Comment by guardrex** [comment]
@tdykstra u free to put an :eye: on this?

**Pull request merged**